pkg/extensions/search/cve/trivy/scanner.go

```diff
@@ -431,6 +431,7 @@ func (scanner Scanner) scanManifest(ctx context.Context, repo, digest string) (m
 					Reference:   getCVEReference(vulnerability.PrimaryURL, vulnerability.References),
 					Severity:    convertSeverity(vulnerability.Severity),
 					PackageList: newPkgList,
+					PackagePath: vulnerability.PkgPath,
```

Comment:
Note: As mentioned in the issue, it is possible for multiple packages in an image to be impacted by the same CVE. Per CVE, the Package Path needs to be a list to indicate "all impacted paths".

Reply:
According to https://github.com/aquasecurity/trivy/commit/1f0d6290c33c95d5f213cc409a0ff6a53a2c888e, the `PkgPath` is part of the Package Details. This data has been moved to be part of the Package List instead.